CONTRIBUTING.md

```diff
@@ -117,7 +121,31 @@ npm run refresh-package-lock-files
 # Finally, check that the dependency in question has been updated in `packages/shared-metrics/package-lock.json` to the desired version.
 ```
 
-Note: The intent of the procedure above is satisfy security scanners (`npm audit`, Dependabot etc.). `package-lock.json` files are never included when publishing to the npm registry. Thy are only relevant when you execute an `npm install` in _this repository_. Users of `@instana` npm packages will have their own `package-lock.json` or `yarn.lock` file for their application, those are completely independent from the `package-lock.json` files in this repository. Thus, they need to take care of updating their transitive dependencies themselves. We only can make sure that the version ranges we use for our direct dependencies allow to install a dependency tree that has no known vulnerabilities.
+Note: The intent of the procedure above is satisfy security scanners (`npm audit`, Dependabot etc.) we run ourselves. The `package-lock.json` files are never included when publishing to the npm registry, see section <#package-lock.json>.
+
+### Removing A Package Dependency
+
+Do *not* run `npm uninstall ${dependency-name}` in the directory of a package. This will mess up the `package-lock.json` file in that package.
+
+There is no `lerna` command to remove a dependency, that is, there is no `lerna remove --scope` counterpart for `lerna add --scope`. To remove a dependency from a package, remove the corresponding entry from the `dependencies`/`devDependencies`/... section of the package's `package.json` file and then run `npm run refresh-package-lock-files` afterwards. This will remove the package from the `node_modules` folder and also update the `package-lock.json` file correctly.
+
+#### Version Ranges vs. Pinning a Specific Version
```

Comment:
This section is also new. It is one half of https://instana.kanbanize.com/ctrl_board/56/cards/120539/details/ (without actually changing all dependencies from pinnned versions ranges, that needs to be done later)